test/integration/generated/body-array/Source/Generated/AutoRestSwaggerBatArrayClient.swift

```diff
@@ -24,12 +24,12 @@ extension CharacterSet {
 public final class AutoRestSwaggerBatArrayClient: PipelineClient {
     /// API version of the  to invoke. Defaults to the latest.
     public enum ApiVersion: String {
-        /// API version "1.0.0"
-        case v100 = "1.0.0"
+        /// API version ""
+        case v = ""
 
         /// The most recent API version of the
         public static var latest: ApiVersion {
-            return .v100
+            return .v
```

Comment:
So, does this not have an api version for the Swagger itsefl? If so, we should omit this entirely as described: https://github.com/Azure/autorest.swift/issues/192

Reply:
Yes. the testserver swaggers do not have apiVersion defined in global parameters. They only have apiversion defined in each operation. That was why I had the logic to look for apiVersion in operation in earlier iteration. But I have removed that in latest iteration.

Regarding #192, I did try to remove blank api version in generated code when first working on this PR.
But the code won't compile because of ClientOptions in AzureCore expects apiVersion in the protocol.

https://github.com/Azure/azure-sdk-for-ios/blob/50fbd64b19d50a40ecd045ee623cb48e1939e45f/sdk/core/AzureCore/Source/Pipeline/PipelineClient.swift

![image](https://user-images.githubusercontent.com/8701731/100009870-49155580-2d84-11eb-9716-5f0074fccbde.png)